creation/web_base/glidein_startup.sh

```diff
@@ -1022,17 +1026,32 @@ set_proxy_fullpath() {
 
 [ -n "${X509_USER_PROXY}" ] && set_proxy_fullpath
 
-for tk in $(pwd)/*idtoken; do
+num_gct=0
+
+for tk in "$(pwd)/credential_"*".idtoken"; do
+  echo "Setting GLIDEIN_CONDOR_TOKEN to ${tk} " 1>&2
+  num_gct=$(( num_gct + 1 ))
   export GLIDEIN_CONDOR_TOKEN="${tk}"
-  if fullpath="$(readlink -f $tk)"; then
-     echo "Setting GLIDEIN_CONDOR_TOKEN $tk to canonical path ${fullpath}" 1>&2
+  fullpath="$(readlink -f "${tk}" )"
```

Comment:
`glidein_paths.source` contains `robust_realpath()` that is more robust but here readlink should be sufficient
Include it directly in the if and `$()` does not require quotes, it protects spaces, no tokenization:
```
if fullpath=$(readlink -f "${tk}" ) ; then
```